src/common/utils/time-monitor/time-monitor.ts

```diff
@@ -38,6 +38,12 @@ export class TimeMonitor {
     setInterval(this.doChecks, 1000);
   }
 
+  removeCheck(name: string): this {
+    this.checks[name] = null;
```

Comment:
Maybe we should make checks more feasible data structure? `Map` or `Set`? With meaningful deletion, so we won't have `!check[name]` condition ...

Reply:
good point, ill change it